app/controllers/users_controller.rb

```diff
@@ -1,5 +1,11 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @user_posts = Post.where(author_id: @user.id)
+    @recent_posts = @user.get_recent_posts
+  end
```

Comment:
- [REQUIRED] Good job so far! The `users_controller` encountered a problem while running your project, please fix it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/232038620-dbdaec0a-c919-412e-92d6-bf64d4f2fb4a.png)